### ISSUE TITLE
For those that dont have xCode 

### ISSUE THREAD

**Issue by AstroOrbis** [ryanslikesocool/Unity-Hub-Swift#1]
For those people that do not have xCode or xCodeTools installed (criiinge), can you put a .app release? I’ll make one in a PR

**Comment by ryanslikesocool** [comment]
I'm currently rewriting and fixing a lot of things on the dev branch, so as soon as that's done, I'll go ahead and upload a build.

**Comment by AstroOrbis** [comment]
If you want i'll compile the build

**Comment by ryanslikesocool** [comment]
No, thats alright.  I'll be changing too much for a build to be worthwhile while I'm moving things around.  I appreciate the offer though!

**Comment by AstroOrbis** [comment]
no problemo :)

**Comment by ghost** [comment]
Hi, I work with AstroOrbis who's been commenting... him and I would like to reuse this code for an aspect of a new app we are working on. Would that be okay?

**Comment by AstroOrbis** [comment]
...Basically just use the sidebar and the file detection script :p

**Comment by ghost** [comment]
Yeah! It would be really helpful, if that is possible, it would save us a lot of time...

**Comment by ryanslikesocool** [comment]
Feel free to use any project assets in accordance with the license.
